packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -370,15 +400,19 @@ export interface Corpus<ChunkMetadata extends Jsonifiable = Jsonifiable> {
   startLoading: () => Promise<Corpus.Stats>;
```

Comment:
Can we get rid of this and just `await corpus.waitForLoadingToComplete` (or maybe `await corpus.createIndex()`)

Reply:
WDYT of this?